test/Benchmarks/Metrics/HistogramBenchmarks.cs

```diff
@@ -49,12 +62,12 @@ namespace Benchmarks.Metrics
     public class HistogramBenchmarks
     {
         private const int MaxValue = 1000;
-        private static readonly ThreadLocal<Random> ThreadLocalRandom = new(() => new Random());
-        private Histogram<long> histogramLong;
-        private Histogram<double> histogramDouble;
+        private Random random = new Random();
```

Comment:
Why make this not `ThreadLocal`? I was recommended to use it by @reyang due to potential very bad behaviour when using the same `Random` instance concurrently in a multi-threaded application.
Although, by default `benchmarkdotnet` uses a single thread.